src/EnergyPlus/ChillerGasAbsorption.cc

```diff
@@ -813,30 +747,28 @@ namespace ChillerGasAbsorption {
 			Node( GasAbsorber( ChillNum ).HeatSupplyNodeNum ).TempSetPointLo = Node( PlantLoop( GasAbsorber( ChillNum ).HWLoopNum ).TempSetPointNodeNum ).TempSetPointLo;
 		}
 
-		if ( ( GasAbsorber( ChillNum ).isWaterCooled ) && ( ( GasAbsorber( ChillNum ).InHeatingMode ) || ( GasAbsorber( ChillNum ).InCoolingMode ) ) && ( ! MyPlantScanFlag( ChillNum ) ) ) {
+		if ( ( GasAbsorber( ChillNum ).isWaterCooled ) && ( ( GasAbsorber( ChillNum ).InHeatingMode ) || ( GasAbsorber( ChillNum ).InCoolingMode ) ) && ( !Init_MyPlantScanFlag( ChillNum ) ) ) {
 			mdot = GasAbsorber( ChillNum ).DesCondMassFlowRate;
 			//DSU removed, this has to have been wrong (?)  Node(CondInletNode)%Temp  = GasAbsorber(ChillNum)%TempDesCondReturn
 
 			SetComponentFlowRate( mdot, GasAbsorber( ChillNum ).CondReturnNodeNum, GasAbsorber( ChillNum ).CondSupplyNodeNum, GasAbsorber( ChillNum ).CDLoopNum, GasAbsorber( ChillNum ).CDLoopSideNum, GasAbsorber( ChillNum ).CDBranchNum, GasAbsorber( ChillNum ).CDCompNum );
 
 		} else {
 			mdot = 0.0;
-			SetComponentFlowRate( mdot, GasAbsorber( ChillNum ).CondReturnNodeNum, GasAbsorber( ChillNum ).CondSupplyNodeNum, GasAbsorber( ChillNum ).CDLoopNum, GasAbsorber( ChillNum ).CDLoopSideNum, GasAbsorber( ChillNum ).CDBranchNum, GasAbsorber( ChillNum ).CDCompNum );
+			if ( GasAbsorber( ChillNum ).CDLoopNum > 0 ){
```

Comment:
It's just odd to be checking that, but I see now that similar if *LoopNum > 0 lines appear in many places in this chiller model.  That seems to indicate a flaw in the init logic that allows these blocks to be executed before the loopnum has been found or is executing them for the wrong operating mode.  I don't see ifs like this in ChillerElectricEIR so something has been handled differently there.  Given that this pattern already exists in this component, then I'll let it stand and move on.